provider/swarm.go

```diff
@@ -39,7 +39,10 @@ func (p SwarmProvider) Name() string {
 
 func (p SwarmProvider) Scale(serviceId string, target int, direction bool) error {
 	ctx := context.Background()
-	service, _, err := p.dockerClient.ServiceInspectWithRaw(ctx, serviceId)
+	// Correct bug #41 using docker/docker v17.06.1-ce-rc4
+	siopts := types.ServiceInspectOptions{}
+	siopts.InsertDefaults = true
```

Comment:
Please add a comment like:

```
// Service inspect returns a service showing default values in empty fields
```

I had to actually execute the code to understand how the struct was filled, I don't want to do that too in the future :D